src/main/java/net/atos/zac/zoeken/model/zoekobject/DocumentZoekObject.java

```diff
@@ -358,6 +358,7 @@ public EnumSet<DocumentIndicatie> getDocumentIndicaties() {
             return EnumSet.noneOf(DocumentIndicatie.class);
         }
         return this.indicaties.stream().map(DocumentIndicatie::valueOf)
+                .filter(indicatie -> !indicatie.equals(DocumentIndicatie.GEBRUIKSRECHT))
```

Comment:
PLaats het verwijderen van de Gebruiksrecht indicatie in de converter waar je ook al het setten van restDocumentZoekObject.indicatieGebruiksrecht verwijderd hebt. De regel hierboven mag dan weer weg.
Zo staat het allemaal wat dichter bij elkaar.

Reply:
Verplaatst naar converter in nieuwe commit